### PR TITLE
I1464 machine readable output

### LINKED ISSUE #1464: We require a machine readable output example document
[kind/Documentation]

## What did you find missing in the documentation?

No examples on machine readable output.

## What is the relevance of it?

There needs to be a guide that outlines all machine readable output options so that projects such as IDE's / VSCode / other projects using it will have a nice reference to look at / implement their code instead of guessing. 

This document should outline command-line examples and output.

### PR DESCRIPTION
**Pull request by boczkowska**
## What is the purpose of this change? What does it change?
List of outputs for json commands

## Was the change discussed in an issue?
fixes #1464

### CODE REVIEW

**Comment by cdrage** [comment]
Yup. This all looks good to me @boczkowska !

File: machine-output.md [hunk 1]
```diff
@@ -0,0 +1,306 @@
...
+      }
+    }
+  ]
+}
+```
+
+
+`odo app list --project myproject -o json | jq`
+ 
+```json
```
Comment (surajnarwade) [inline machine-output.md]: this output is empty, can you please check once again ?

**Comment by amitkrout** [comment]
@cdrage @surajnarwade I am concern about the commit location of the file. I feel the best commit place would in ```doc/ ``` dir. WDYT ?

**Comment by kadel** [comment]
`/docs` would be a better location for this file. Otherwise, this looks good

**Comment by amitkrout** [comment]
/approve
@cdrage @surajnarwade ping

**Pull request merged**